filesmanager/filesmanager.py

```diff
@@ -19,6 +19,8 @@
 from xblock.fragment import Fragment
 from xblockutils.resources import ResourceLoader
 
+from filesmanager.processors.xapi.event_transformers.filesmanager_events import \
+    FilesDownloadedTransformer  # pylint: disable=unused-import
```

Comment:
```suggestion
from filesmanager.processors.xapi.event_transformers import FilesDownloadedTransformer # pylint: disable=unused-import
```

This is possible since you are importing that in the __init__.py file